test/message/test-testpy-env-var-via-comment.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+// Environment Variables: A_SET_ENV_VAR=A_SET_ENV_VAR_VALUE B_SET_ENV_VAR=B_SET_ENV_VAR_VALUE
```

Comment:
Just a nit but I'd prefer the more compact: `// Env: ...`

Reply:
+1 (@joyeecheung suggested the same name)
I’ll go ahead and update the PR

Reply:
done!